### PR TITLE
Explain how to amend a proposal

### PR DESCRIPTION
**Pull request by nomeata**
as asked for in #296

### CODE REVIEW

**Comment by andrewthad** [comment]
Perfect. It's terse and includes the word "amend", which makes it easy for anyone looking for those instructions to Control+F for it.

**Comment by nomeata** [comment]
@simonpj (as the chair) do you approve this wording?

File: README.rst [hunk 1]
```diff
@@ -142,6 +142,16 @@ in your branch. For instance,
...
     [Rendered](https://github.com/bgamari/ghc-proposals/blob/typeable/proposals/0000-type-indexed-typeable.rst)
 
+How to amend an accepted proposal
+---------------------------------
+
+
+Some proposals amend an existing proposal, i.e. make a change that is more than
+editorial in nature (and warrant approval by the committee), but are too small or
+too much tied to the existing proposal to be their own proposal. Often, this happens
+after a proposal is accepted, but before or while it is implemented.
```
Comment (simonpj) [inline README.rst]: Slightly hard to parse.  I'd say
```
Some proposals amend an existing proposal.   Such an ammendment :
* Makes a significant (i.e. not just editorial or typographical) change, and hence warrants approval by the commitee
* Is too small, or too closely tied to the existing proposal, to make sense as a new standalone proposal.
Often this happens...
```
*

**Comment by simonpj** [comment]
I've suggested an editorial change but am happy with the intent
